src/pages/docs/creators/python/tutorials/task-example-0-hello.md

```diff
@@ -0,0 +1,301 @@
+)
+```
+
+Currently, Golem is using a public repository to store both official and community-authored VM images. Once an image is uploaded to this repository it can be referred to by its hash.
+
+This is what we're making use of here - by using the function `repo` from `vm`, we're getting a payload definition for our providers. The only input we must provide at this point is the image hash. In the case of this example we're using a pre-uploaded, minimal image based on Alpine Linux.
+
+{% alert level="info" %}
+If you'd like to learn about creating and uploading Golem images yourself, take a look at this guide: [Golem images](/docs/creators/python/guides/golem-images)
+{% /alert %}
+
+### Tasks array
+
+```python
+tasks = [Task(data=None)]
+```
+
+Next comes the array of task fragments to be computed. For simplicity, our `tasks` array contains a single item of type `Task` which has no data associated with it. This means we only need a single item to be computed and that the worker function does not need any additional parameters.
+
+In general, each `Task` object refers to a single piece of computation within your app and typically holds some data. For example, in a program which operates on a huge file, a single `Task` could be holding one chunk of that file to be processed by one of many providers involved.
+
+{% alert level="info" %}
+To see a more involved example of this take a look at: [Task Example 1: Simple hash cracker](/docs/creators/python/tutorials/task-example-1-cracker#the-task-fragments) (this links to a section on task fragments)
+{% /alert %}
+
+### Golem/Executor
+
+```python
+async with Golem(budget=1.0, subnet_tag="public") as golem:
+    async for completed in golem.execute_tasks(worker, tasks, payload=package):
+        print(completed.result.stdout)
+```
+
+Finally, as the last step of our `main()` function we create an instance of `Golem` (or `Executor` in the case of JS API) and use it to request our tasks.
+
+`Golem/Executor` is the heart of Golem's API - it handles communication with the local Golem node, processes payments for completed tasks and, most importantly, provides an easy way of requesting resources in the Golem network.
+
+#### Instantiation
+
+Let's first focus on the instantiation code:
+
+```python
+async with Golem(budget=1.0, subnet_tag="public") as golem:
+    ...
+```
+
+{% alert level="info" %}
+`Golem/Executor` instances work as asynchronous [context managers](https://docs.python.org/3/reference/datamodel.html#context-managers). Since context managers are a concept native to Python, Golem's `yajsapi` provides a custom implementation in the form of the function `asyncWith`.
+
+Context managers are somewhat similar to `try-catch-finally` blocks. They allow for setup and teardown logic before using some resource. In our case this resource is the `Golem/Executor` instance, while the block of code inside `async with/asyncWith` is the context.
+{% /alert %}
```

Comment:
it seems this piece is a remainder of the times when this article was hybrid JS/Python content

```suggestion
```